installer/bundle/bundle_skel.sh

```diff
@@ -625,6 +625,7 @@ cd $EXTRACT_DIR
 # Do we need to remove the package?
 set +e
 if [ "$installMode" = "R" -o "$installMode" = "P" ]; then
+    rm -f /etc/cron.d/OMSConsistencyInvoker
```

Comment:
Can you also add a description to this commit for what issue this fix solves ? Would this change fix the verbose message with files in use warning during uninstall too?